study-builder/src/main/java/org/broadinstitute/ddp/studybuilder/StudyBuilderContext.java

```diff
@@ -0,0 +1,38 @@
+package org.broadinstitute.ddp.studybuilder;
+
+import static org.broadinstitute.ddp.studybuilder.translation.TranslationsUtil.TRANSLATION_KEY_PREFIX;
+
+import java.util.Map;
+import java.util.Properties;
+
+import com.typesafe.config.Config;
+import org.broadinstitute.ddp.studybuilder.translation.TranslationsUtil;
+
+
+public class StudyBuilderContext {
+
```

Comment:
This class holds the translations fetched from translation files (it is required during translations references auto-resolving).